lib/cli_common/cli_common/utils.py

```diff
@@ -36,5 +36,5 @@ def __exit__(self, *args):
         except Exception as e:
             for future in self.futures:
                 future.cancel()
-            raise
+            raise e
```

Comment:
what would be the exception to re-raise? i thought this is `raise e`

Reply:
If you do `raise e`, Python creates a new exception modifying the traceback, instead if you just do `raise`, it will re-raise the previous exception, with the original traceback

Reply:
oh, i see, then we need to remove that `as e`